src/bench/bench.cpp

```diff
@@ -20,7 +20,15 @@ const std::function<void(const std::string&)> G_TEST_LOG_FUN{};
 
 void benchmark::ConsolePrinter::header()
 {
-    std::cout << "# Benchmark, evals, iterations, total, min, max, median" << std::endl;
+    std::cout << "#"
+              << std::left << std::left << std::setw(47) << "Benchmark"
```

Comment:
I played a bit around with it and I find it better to change this to `16`. This will be much more friendly on smaller terminals and/or display

Reply:
Well that doesn't work with tests like `CHACHA20_POLY1305_AEAD_256BYTES_ENCRYPT_DECRYPT`.
see the last paragraph in my post

Reply:
Couldn't this be dynamically calculated then?

Reply:
>  this is limited by up to 47 chars of benchmark name, so as long as we don't add more names like CHACHA20_POLY1305_AEAD_256BYTES_ENCRYPT_DECRYPT and longer then it will be fine.
(it can probably be adjustable but that will require iterating over all the tests before running them to determine the longest cell and I thought the 47 limit is more than reasonable)